README.md

```diff
@@ -50,14 +50,14 @@ installation steps should be completed before the following steps, which are spe
 
 #### Create your docker container:
 
-The following commands create a Docker machine named ``open_discussions``, start the
+The following commands create a Docker machine named ``opendiscussions``, start the
 container, and configure environment variables to facilitate communication
 with the edX instance.
 
-    docker-machine create --driver virtualbox open_discussions
-    docker-machine start open_discussions
+    docker-machine create --driver virtualbox opendiscussions
+    docker-machine start opendiscussions
     # 'docker-machine env (machine)' prints export commands for important environment variables
-    eval "$(docker-machine env open_discussions)"
+    eval "$(docker-machine env opendiscussions)"
```

Comment:
I don't think this will cause a problem, it's mostly an issue if you wanted to destroy one VM for some reason without affecting the others. Anyway let's leave this as is, I'll see about fixing it in cookiecutter